hedera-node/docs/assets/Palantir_Style_SL.xml

```diff
@@ -0,0 +1,576 @@
+<code_scheme name="Palantir Style (SL)" version="173">
+  <option name="OTHER_INDENT_OPTIONS">
+    <value>
+      <option name="INDENT_SIZE" value="2" />
```

Comment:
indent should be 4.
tab size should _probably_ be 4 to match (although I find making it 8 helps highlight when tabs are used when they shouldn't be).

Reply:
@jsync-swirlds This is the current code style which matches the Palantir formatting and as far as I can tell generates the proper indentations.

Reply:
Spotless will kick out the indentation (which definitely doesn't match what's here).
Spotless is 4 space indentation, not 2.

Reply:
hmm.  I downloaded these two files (style + copyright) from our internal [notions' developer setup page](https://www.notion.so/swirldslabs/008-Java-Code-Style-IntelliJ-IDE-Configuration-41098cd7f42b4b39bfdedc0995bd071a?pvs=4) so it's wrong there?

Reply:
The file is incorrect on notion, yes.  As far as I can tell it was initially and has remained so.